README.md

```diff
@@ -220,37 +224,41 @@ the file are.*
 Inspects your Flask app's and blueprint's `static_folder` and uses that as the
 input path of where to look for digested and compressed files.
 
-It will recursively delete files that have a file extension of `.gz` and also
-deletes files that have been digested. It determines if a file has been
-digested based on its file name. In other words, it will delete files that
-match this regexp `r"-[a-f\d]{32}"`.
+It will recursively delete files that have a file extension of `.gz` or `.br`
+and files that have been digested. It determines if a file has been digested
+based on its file name. In other words, it will delete files that match this
+regexp `r"-[a-f\d]{32}"`.
 
 In the end that means if you had these 4 files in your static folder:
 
 - `images/flask.png`
 - `images/flask.png.gz`
 - `images/flask-f86b271a51b3cfad5faa9299dacd987f.png`
 - `images/flask-f86b271a51b3cfad5faa9299dacd987f.png.gz`
+- `images/flask-f86b271a51b3cfad5faa9299dacd987f.png.br`
 
 And you decided to run the clean command, the last 3 files would be deleted
 leaving you with the original `images/flask.png`.
 
 ## Configuring this extension
 
 By default this extension will md5 tag all files it finds in your configured
-`static_folder`. It will also create gzipped versions of each file and it won't
-prefix your static files with an external host.
+`static_folder`. It will also create gzip'ed versions of each file and it
+won't prefix your static files with an external host.
 
-If you don't like any of this behavior, you can optionally configure:
+If you don't like any of this behavior, or you wish to enable brotli, you can
+optionally configure:
 
 ```py
 FLASK_STATIC_DIGEST_BLACKLIST_FILTER = []
 # If you want specific extensions to not get md5 tagged you can add them to
 # the list, such as: [".htm", ".html", ".txt"]. Make sure to include the ".".
 
-FLASK_STATIC_DIGEST_GZIP_FILES = True
-# When set to False then gzipped files will not be created but static files
+FLASK_STATIC_COMPRESSION = ["gzip", "brotli"]
```

Comment:
Let's default to `FLASK_STATIC_COMPRESSION = ["gzip"]`

With a description of:

```
# Optionally compress your static files, supported values are:
#   []                 avoids any compression
#   ["gzip"]           uses gzip
#   ["brotli"]         uses brotli
#   ["gzip", "brotli"] uses both
```